docs/ops/movement/StridedSlice_1.md

```diff
@@ -5,37 +5,37 @@
 **Category**: Data movement operation
 
 **Short description**: *StridedSlice* extracts a strided slice of a tensor.
- It is similar to generalized array indexing in Python\*.
```

Comment:
Not sure that this change is correct. @ntyukaev, @tsavina could you comment on that?

Reply:
In general I did review this spec regarding *Types* section and it's usage + some minor styling improvements. This statement was already in spec, and as it was already reviewed before I assume it is correct. The change here is only to remove \* characters which seemed unneeded.

Reply:
@ntyukaev @tsavina "_It is similar to generalized array indexing in Python_" is this statement correct or should it be removed?

Reply:
It looks like this statement is borrowed from [tensorflow](https://www.tensorflow.org/api_docs/python/tf/strided_slice). I removed it in this [commit](https://github.com/openvinotoolkit/openvino/pull/5887/commits/d93345afd87b3d8a2fff4ac24ed02f17b7bd95d6) as it may bring more confusion than value.